config_defaults_inc.php

```diff
@@ -5117,3 +5117,15 @@
  * the attachment is linked to the note.  Or 0 for disabling this feature.
  */
 $g_issue_activity_note_attachments_seconds_threshold = 3;
+
+#################################
+# Simple Report Screen settings #
+#################################
+
+/**
+ * Define the max Level Treshold  for users to present the simple entry screen
+ * Define the fileds to be displayed ( initial value is same as current set
+ * example : $g_simple_bug_report_page_fields = array(	'additional_info',	'attachments',	'category_id',);
+ */
+ g_simple_report_level = UPDATER;
```

Comment:
The build is broken because the missing "$" introduces a syntax error.

Reply:
Thanks for correcting the typo

Reply:
@CasN I did not correct anything. This was just one of these typical notes you will get during PR / code review in GitHub.
You have to correct it in your branch and push it, so that the PR gets updated.

Reply:
I did correct om my side, followed by a push but I do not see the change within the pull request yet. How much time can be in between?

Reply:
It typically takes just a few seconds.
I don't see any new change since yesterday on your clone in branch Simple-report-screen https://github.com/CasN/mantisbt/tree/Simple-report-screen

Reply:
OK, added a second commit in order to fix the missing character, hope i got it right now